staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -583,6 +584,18 @@ func installAPI(s *GenericAPIServer, c *Config) {
 			routes.DefaultMetrics{}.Install(s.Handler.NonGoRestfulMux)
 		}
 	}
+
+	// so far, only logging related endpoints are considered valid to add for these debug flags.
```

Comment:
I feel like we should flag guard this somehow, include it in one of the above or add a new one?

Reply:
Could do this as a follow up.

Reply:
Some system administrators really hate it if users can get to these debug endpoints. We need some way to make sure non-authorized users can't call it, or the ability to turn it off completely. It can be blocked in the same way the other debug endpoints are: please move this into the `if c.EnableProfiling` block.

Reply:
e.g., setting v=9 is a good way to make a slightly underprovisioned apiserver unresponsive, I bet.

Reply:
This is supposed be controllable with RBAC, with the non-resource URLs in a role. But I agree that not even this is wanted in some environments (like probably GKE).

Reply:
>please move this into the if c.EnableProfiling block.

@lavalamp done